Dockerfile

```
@@ -0,0 +1,22 @@
+FROM rust:alpine as builder
+
+RUN apk add --no-cache musl-dev openssl-dev
+
+RUN rustup default stable
+RUN USER=root cargo new --bin lutgen
+
+WORKDIR /lutgen
+
+COPY . .
+
+RUN RUSTFLAGS="-Ctarget-feature=-crt-static" cargo build --release
+
+FROM alpine:latest
+
+RUN apk add --no-cache gcc
+RUN adduser -D -s /bin/sh lutgen
+USER lutgen
+
+COPY --from=builder /lutgen/target/release/lutgen /lutgen
```

```suggestion
COPY --from=builder /lutgen/target/release/lutgen /lutgen
COPY --from=builder /lutgen/target/release/lutgen-studio /lutgen-studio
```

Might as well include it since we're building the whole workspace. May need to also install the runtime deps for studio to work as well (xkbcommon, xcb*, openssl). Alternative could be to pass `--bin lutgen` to cargo build and only build the cli though, might be neater.

---

Oh sorry, yeah, I should just build the cli only, I will modify that ASAP. Thanks for noticing.